src/graphql/resolvers.js

```diff
@@ -153,10 +153,30 @@ module.exports = function Resolvers (ssb) {
    * @param {int} opts.threadMaxSize - max amount of messages in each thread to return
    */
   const getThreads = (feedId, opts) => {
-    const { threadMaxSize, limit = 10 } = opts
+    const { threadMaxSize, limit = 10, cursor } = opts
+
+    var keepSkipping = Boolean(cursor)
+
+    const isByMember = (thread) => thread.messages.some(message => message?.value.author === feedId)
+    const handleSkip = (thread) => {
+      const id = thread.messages[0].key
+      if (id === cursor) keepSkipping = false
+      return false
+    }
+
     return new Promise((resolve, reject) => {
       pull(
         ssb.threads.profile({ id: feedId, reverse: true, threadMaxSize, allowlist: ['post'] }),
+
+        pull.filter(thread => {
+          if (!isByMember(thread)) return false
+
+          // only skip when need to
+          // this is a naive approach to pagination
+          // by skipping items when there is a cursor
+          if (!keepSkipping) return true
+          return handleSkip(thread)
```

Comment:
this will work but will get increasingly slow as you load more stuff right? as you'll load threads you already have and are skipping.
I would put a `HACK` note in, and then 
`TODO - add pagination to ssb-threads`